lib/activeadmin_settings_cached/dsl.rb

```diff
@@ -32,6 +33,7 @@ def active_admin_settings_page(options = {}, &block)
 
         flash[:success] = t('activeadmin_settings_cached.settings.update.success'.freeze)
         Rails.version.to_i >= 5 ? redirect_back(fallback_location: admin_root_path) : redirect_to(:back)
+        options[:update_callback].call if options[:update_callback].respond_to?(:call)
```

Comment:
May be it be more beter if we name this option `after_save`
And can u add test with real behavior to see case when it needed?